src/network/netdev/bond.c

```diff
@@ -414,12 +419,14 @@ int config_parse_ad_user_port_key(const char *unit,
 
         r = safe_atou16(rvalue, &v);
         if (r < 0) {
-                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse user port key '%s', ignoring: %m", rvalue);
+                log_syntax(unit, LOG_ERR, filename, line, r,
+                           "Failed to parse user port key '%s', ignoring: %m", rvalue);
                 return 0;
         }
 
         if (v > 1023) {
-                log_syntax(unit, LOG_ERR, filename, line, 0, "Failed to parse user port key '%s'. Range is [0,1023], ignoring.", rvalue);
+                log_syntax(unit, LOG_ERR, filename, line, 0,
+                           "Failed to parse user port key '%s'. Range is [0,1023], ignoring.", rvalue);
```

Comment:
not that it matters at all, but `[0…1023]` is sooo much nicer. unicode!

Reply:
Yeah!